pwndbg/commands/start.py

```diff
@@ -9,25 +9,33 @@
 from argparse import RawTextHelpFormatter
 from shlex import quote
 
-import gdb
-
 import pwndbg
+import pwndbg.aglib.elf
+import pwndbg.aglib.proc
+import pwndbg.color.message as M
 import pwndbg.commands
-import pwndbg.gdblib.elf
-import pwndbg.gdblib.symbol
+import pwndbg.dbg
 from pwndbg.commands import CommandCategory
-from pwndbg.dbg import EventType
+from pwndbg.dbg import BreakpointLocation
+
+if pwndbg.dbg.is_gdblib_available():
+    import gdb
```

Comment:
I don't really love that with `@OnlyWithDbg` we still need to do the whole song and dance of gating off imports depending on the debugger, but It works well enough, and I think it's a good enough solution, all things considered, so I guess it's fine?

Reply:
Why do we actually need import gdb there?

Reply:
Perhaps we can do `if gdb := pwndbg.dbg.get_gdb()`?

Reply:
I tried do `get_gdb` by `mypy` complain